scripts/r.in.aster/r.in.aster.py

```diff
@@ -133,20 +149,25 @@ def main():
         "13",
         "14",
     ]
+    
+    # Band 3b is not included ASTER L1T
+    if proctype == 'L1T':
+        allbands.pop(3)
```

Comment:
```suggestion
        allbands.remove("3b")
```

Selfdocumenting and no need to keep in sync with construction of the variable `allbands` (if there ever will be changes).